README.md

```
@@ -0,0 +1,263 @@
+
+## Development
+
+### Building
+
+```bash
+npm run build
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)